### PR TITLE
Enabled snupkg as a Symbol package format.

### PR DESCRIPTION
**Pull request by Aleh-Yanushkevich**
- Removed *.props and *target files.
- Enabled snupkg as a symbol package format by default.

### CODE REVIEW

File: src/ReportPortal.Client/ReportPortal.Client.csproj [hunk 1]
```diff
@@ -12,9 +12,10 @@
     <RepositoryType>GitHub</RepositoryType>
     <RepositoryUrl>https://github.com/reportportal/client-net</RepositoryUrl>
     <PackageTags>report reportportal api client testing</PackageTags>
-	  <PackageReadmeFile>README.md</PackageReadmeFile>
-	  
-    <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
+    <PackageReadmeFile>README.md</PackageReadmeFile>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <IncludeSymbols>true</IncludeSymbols>
+    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
```
Comment (nvborisenko) [inline src/ReportPortal.Client/ReportPortal.Client.csproj:18]: It's already by default if I ma not mistaken.

**Review by nvborisenko — CHANGES_REQUESTED** [review]

**Comment by nvborisenko** [comment]
And we need to save `snupkg` as artifact in build pipeline.

File: src/ReportPortal.Client/ReportPortal.Client.csproj (hunk 1 above)
Comment (Aleh-Yanushkevich) [reply src/ReportPortal.Client/ReportPortal.Client.csproj:18]: Sure! Forgot about it, thank you!

**Review by nvborisenko — APPROVED** [review]

**Comment by Aleh-Yanushkevich** [comment]
> And we need to save `snupkg` as artifact in build pipeline.

Sure! Forgot about it, thank you!

**Pull request merged**